### PR TITLE
Price Denomination derived from other feeds

### PR DESCRIPTION
**Pull request by melchermaxwell**
initial commit
Added conversion formula.  .DS_Store to gitignore
Fixed typo and added using SafeMath
Edited the example to be in BTC not ETH

### CODE REVIEW

File: docs/Using Price Feeds/data-feeds-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+permalink: 'docs/data-feeds-best-practices/'
+---
+
+Below are some of the practices for using Chainlink Data Feeds.
+
+## Getting a different price denomination
+
+Chainlink Price Feeds can be used in combination to derive denominated price pairs in other currencies.
+
+In the event you require a denomination other than what is provided, two price feeds can be used to derive the pair you need. For example, if you needed an BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using <a href="https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol" target="_blank">OpenZeppelin's SafeMath</a> division.
```
Comment (dwightjl) [inline docs/Using Price Feeds/data-feeds-best-practices.md:15]:
```suggestion
If you require a denomination other than what is provided, you can use two price feeds to derive the pair that you need. For example, if you needed an BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol) division.
```

File: docs/Using Price Feeds/data-feeds-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+![Request Model Diagram](/files/price-feed-conversion-equation.gif)
```
Comment (dwightjl) [inline docs/Using Price Feeds/data-feeds-best-practices.md:17]: Please put this in `_src/images/` if you can. Sorry it's confusing that we have two different folders with images.

File: _data/navigation.js [hunk 3]
```diff
@@ -209,6 +209,10 @@ module.exports = {
             title: 'API Reference',
             url: '/docs/price-feeds-api-reference/',
           },
+          {
+            title: 'Best Practices',
```
Comment (dwightjl) [inline _data/navigation.js:213]:
```suggestion
            title: 'Best Practices for Data Feeds',
```

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,5 @@ _site/
 node_modules
 .firebase/
 data-source/
-addresses.json
\ No newline at end of file
+addresses.json
+.DS_Store
```
Comment (dwightjl) [inline .gitignore:6]: I'm not seeing this in my hidden folder list. Is this necessary?

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 4 above)
Comment (melchermaxwell) [reply .gitignore:6]: @dwightjl I can remove if you'd prefer.

Comment (dwightjl) [reply .gitignore:6]: LGTM thanks for clarifying!

**Review by dwightjl — APPROVED** [review]

**Review by dwightjl — APPROVED** [review]
Latest changes LGTM

File: docs/Using Price Feeds/get-the-latest-price.html [hunk 5]
```diff
@@ -89,6 +89,23 @@
     </div>
 </div>
 
+## Getting a different price denomination
+
+Chainlink Price Feeds can be used in combination to derive denominated price pairs in other currencies.
```
Comment (dwightjl) [inline docs/Using Price Feeds/get-the-latest-price.html:94]:
```suggestion
Chainlink Data Feeds for asset prices can be used in combination to derive denominated price pairs in other currencies.
```

File: docs/Using Price Feeds/get-the-latest-price.html [hunk 6]
```diff
@@ -89,6 +89,23 @@
...
+
+If you require a denomination other than what is provided, you can use two price feeds to derive the pair that you need. For example, if you needed an BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol) division.
```
Comment (dwightjl) [inline docs/Using Price Feeds/get-the-latest-price.html:96]:
```suggestion
If you require a denomination other than what is provided, you can use two feeds to derive the pair that you need. For example, if you needed an BTC / EUR price, you could take the BTC / USD feed and the EUR / USD feed and derive BTC / EUR using [OpenZeppelin's SafeMath](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeMath.sol) division.
```

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Review by dwightjl — APPROVED** [review]

**Pull request merged**